src/Tools/dotnet-monitor/Auth/ApiKey/UserAuthorizationHandler.cs

```diff
@@ -36,7 +29,7 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
             if (context.User.Identity.AuthenticationType == AuthConstants.FederationAuthType)
             {
                 // If we get a FederationAuthType (Bearer from a Jwt Token) we need to check that the user has the specified subject claim.
-                string expectedSubjectClaim = _pinnedSubject ?? _apiKeyConfig.CurrentValue.Subject;
+                string expectedSubjectClaim = _apiKeyConfig.CurrentValue.Subject;
```

Comment:
nit: Is there any reason to keep this variable (as opposed to just using the value directly below in the conditional)?